src/cdk-experimental/listbox/listbox.ts

```diff
@@ -400,6 +417,51 @@ export class CdkListbox implements AfterContentInit, OnDestroy, OnInit {
     this._listKeyManager.updateActiveItem(option);
   }
 
+  /**
+   * Saves a callback function to be invoked when the select's value
+   * changes from user input. Required to implement ControlValueAccessor.
+   */
+  registerOnChange(fn: (value: any) => void): void {
+    this._onChange = fn;
+  }
+
+  /**
+   * Saves a callback function to be invoked when the select is blurred
+   * by the user. Required to implement ControlValueAccessor.
+   */
+  registerOnTouched(fn: () => {}): void {
+    this._onTouched = fn;
+  }
+
+  /** Sets the select's value. Required to implement ControlValueAccessor. */
+  writeValue(value: any): void {
+    if (this._options) {
+      this._setSelectionByValue(value);
+    }
+  }
+
+  /** Disables the select. Required to implement ControlValueAccessor. */
+  setDisabledState(isDisabled: boolean) {
+    this.disabled = isDisabled;
+  }
+
+  /** Selects an option that has the corresponding given value. */
+  private _setSelectionByValue(values: any | any[]) {
+    for (const option of this._options.toArray()) {
+      this.deselect(option);
+    }
+
+    for (const value of values) {
+      const correspondingOption = this._options.find((option: CdkOption) => {
+        return option.value != null && this.compareWith(option.value,  value);
+      });
+
+      if (correspondingOption) {
```

Comment:
Should we break the loop once we've found a match in single selection mode?

Reply:
Yeah I can add that.